src/app/doc.cpp

```diff
@@ -528,6 +528,7 @@ void Doc::copyLayerContent(const Layer* sourceLayer0, Doc* destDoc, Layer* destL
       if (it != linked.end()) {
         newCel.reset(Cel::MakeLink(sourceCel->frame(),
                                    it->second));
+        newCel->setZIndex(sourceLayer0->cel(sourceCel->frame())->zIndex());
```

Comment:
This can be:
```
        newCel->setZIndex(sourceCel->zIndex());
```
as `sourceCel` is already equal to `sourceLayer0->cel(sourceCel->frame())`.